app/modules/fcm/controllers/fcmControllers.js

```diff
@@ -84,8 +85,42 @@ angular.module('pcApp.fcm.controllers.fcm', [
 
                 // broadcasting the event
                 $rootScope.$broadcast('appChanged');
+
+                var reqData = {
+                    "body": {
+                        "size": 1000,
+                        "from": 0,
+                        "sort": ["title.lower_case_sort"],
+                        "query": {
+                            "bool": {
+                                "must": [{ "term": { "_type": "fuzzymap" } }],
+                                "should": []
+                            }
+                        }
+                    },
+                    "index": "policycompass_search"
+                };
+
+                angular.forEach($scope.models.model.keywords.toString().split(','), function (item) {
+                    if (reqData.body.query.bool.must.length == 1) {
+                        reqData.body.query.bool.must.push({ "match_phrase": { "keywords": $.trim(item) + ",*" } });
+                        reqData.body.query.bool.must.push({ "match_phrase": { "keywords": "*," + $.trim(item) } });
+                    }
+                    else {
+                        reqData.body.query.bool.should.push({ "match_phrase": { "keywords": $.trim(item) + ",*" } });
+                        reqData.body.query.bool.should.push({ "match_phrase": { "keywords": "*," + $.trim(item) } });
+                    }
+                });
```

Comment:
This is because I am not matching the complete text. As you know, users can enter multiple keywords.